.pre-commit-config.yaml

```
@@ -2,7 +2,7 @@
 repos:
 
 # Automatically sort imports with isort
-- repo: https://github.com/timothycrosley/isort.git
+- repo: https://github.com/PyCQA/isort.git
```

```suggestion
- repo: https://github.com/PyCQA/isort
```
messes with renovate